### ISSUE TITLE
Narrator/NVDA is not reading the short keys inside the command palette

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/monaco-editor#1156]
**Environment Details:**
https://docs.microsoft.com/en-us/dotnet/csharp/quick-starts/hello-world?tutorial-step=4User
**Browser:** Edge (41.16299.402.0)
Chrome (67.0.3396.87)
**Screen Reader:** Narrator
NVDA

**Additional Details:**
MAS Violated: MAS1.3.1

**Repro steps:**
1. Hit the URL https://docs.microsoft.com/en-us/dotnet/csharp/quick-starts/hello-world?tutorial-step=4User
2. Tab till “Enter focus mode” and press enter
3. Press F1 and open the command palette
4. Verify Whether the screen reader is reading the short keys and check whether it is accessible by the keyboard.

**Actual Result:**
Screen reader is not reading the short keys inside the command palette

**Expected Result:**
Screen reader should read the short keys for the user who wants to access it

**User Impact:**
Screen readers users find difficult in accessing the keyboard short keys if screen reader does not mention them

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

[screen reader is not reading the short keys inside the command palette.pptx](https://github.com/Microsoft/monaco-editor/files/2529197/screen.reader.is.not.reading.the.short.keys.inside.the.command.palette.pptx)

**Comment by bpasero** [comment]
via https://github.com/Microsoft/vscode/commit/637604810f2913d56a9960d6eaf9711243f89d7a
